docker/ctAutocomplete/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM node:17-alpine
+
+LABEL maintainer="Bernd van der Veen <bernd@ctcue.com>"
+
+USER root
+
+# Install all dependencies
+RUN apk update && apk add tzdata
```

Comment:
I don't think we use `tzdata` for autocompletion... we have zero date or time dependencies. 

Maybe update the comment: "To set the correct timezone, we install some dependencies, they are removed later on"